src/plugins/intel_gpu/include/intel_gpu/runtime/debug_configuration.hpp

```diff
@@ -151,6 +152,7 @@ class debug_configuration {
     bool is_layer_for_dumping(const std::string& layerName, bool is_output = false, bool is_input = false) const;
     bool is_target_iteration(int64_t iteration) const;
     std::string get_matched_from_filelist(const std::vector<std::string>& file_names, std::string pattern) const;
+    bool is_matched_layer(const std::string& layer_name, const std::string& pattern) const;
```

Comment:
This function is actually not about layer but about name. So `is_name_matched` would be better name.

Reply:
applied